lib/matplotlib/inset.py

```diff
@@ -171,7 +176,7 @@ def _update_connectors(self):
                 # parent artist.
                 p = ConnectionPatch(
                     xyA=xy_inset_ax, coordsA=self._inset_ax.transAxes,
-                    xyB=xy_data, coordsB=self.axes.transData,
+                    xyB=xy_data, coordsB=self._rect_trans,
```

Comment:
First I tried just using self.rectangle.get_transform() here, but an existing image test failed.  It seems the transform you get out of a rectangle is not quite what you put in?

Reply:
Do you need to use `get_patch_transform()` instead or subtract that off? There is also mention in comments of needing to add the patches to an Axes before accessing the transforms.

https://github.com/matplotlib/matplotlib/blob/84464dd085210fb57cc2419f0d4c0235391d97e6/lib/matplotlib/patches.py#L307-L309

Reply:
> It seems the transform you get out of a rectangle is not quite what you put in?

It is not; a `Rectangle` is actually a unit square + a transform to the rectangle you really wanted, and then the user transform is added on top of that.

Reply:
Thanks Team Cartopy!  It looks like `get_data_transform` is what I need.